packages/desktop/components/popups/BalanceBreakdownPopup.svelte

```diff
@@ -0,0 +1,167 @@
+
+                let type: string
+                let amount: number
+                if (output.type !== OUTPUT_TYPE_TREASURY) {
+                    if (
+                        output.unlockConditions.some(
+                            (unlockCondition) => unlockCondition.type === UNLOCK_CONDITION_EXPIRATION
+                        )
+                    ) {
+                        type = 'unclaimed'
+                        amount = Number(output.amount)
+                    } else if (
+                        output.unlockConditions.some(
+                            (unlockCondition) => unlockCondition.type === UNLOCK_CONDITION_STORAGE_DEPOSIT_RETURN
+                        )
+                    ) {
+                        type = 'storageDepositReturn'
+                        amount = getStorageDepositFromOutput(output).storageDeposit
+                    } else if (
+                        output.unlockConditions.some(
+                            (unlockCondition) => unlockCondition.type === UNLOCK_CONDITION_TIMELOCK
+                        )
+                    ) {
+                        type = 'timelock'
+                        amount = Number(output.amount)
+                    }
+                }
+
+                if (!subBreakdown[type]) {
+                    subBreakdown[type] = amount
+                } else {
+                    subBreakdown[type] += amount
+                }
+                pendingOutputsStorageDeposit += amount
+            }
+        }
+
+        return { amount: pendingOutputsStorageDeposit, subBreakdown }
+    }
+
+    function getLockedBreakdown(): { amount: number; subBreakdown: Breakdown } {
+        const governanceAmount = parseInt($selectedAccount?.votingPower, 10)
+
+        const totalLockedAmount = governanceAmount
+
+        const subBreakdown = {
+            governance: { amount: governanceAmount },
+        }
+
+        return { amount: totalLockedAmount, subBreakdown }
+    }
+
+    function getStorageDepositBreakdown(): { amount: number; subBreakdown: Breakdown } {
+        const totalStorageDeposit = accountBalance?.requiredStorageDeposit
+            ? Object.values(accountBalance.requiredStorageDeposit).reduce(
+                  (total: number, value: string): number => total + Number(value),
+                  0
+              )
+            : 0
+
+        const subBreakdown = {
+            basicOutputs: { amount: Number(accountBalance?.requiredStorageDeposit?.basic ?? 0) },
+            nftOutputs: { amount: Number(accountBalance?.requiredStorageDeposit?.nft ?? 0) },
+            aliasOutputs: { amount: Number(accountBalance?.requiredStorageDeposit?.alias ?? 0) },
+            foundryOutputs: { amount: Number(accountBalance?.requiredStorageDeposit?.foundry ?? 0) },
+        }
+
+        return { amount: totalStorageDeposit, subBreakdown }
+    }
+
+    function handleConsolidation(): void {
```

Comment:
Clicking this button doesn't do anything for me.

```suggestion
    function onConsolidationClick(): void {
```